api/views.py

```diff
@@ -3025,3 +3034,51 @@ def update_order_param(request):
         else:
             return status_response(False, 'Параметр последний')
     return status_response(True)
+
+
+def get_manage_analyzer(request):
+    analyzers = [{"label": analyzerList.title,
+                  "pk": analyzerList.id} for analyzerList in Analyzer.objects.all().order_by('title', 'id')]
```

Comment:
Нейминг в формате camelCase не надо в питоне использовать. Плюс тут на каждой итерации не список, а отдельные анализаторы, так что:
```suggestion
                  "pk": analyzer.id} for analyzer in Analyzer.objects.all().order_by('title', 'id')]
```

И нехватает фильтрации, что бы убрать анализаторы с пустыми портами и service name